Sigma_meta_filter.md

```diff
@@ -1,4 +1,4 @@
-# Sigma Meta Filter <!-- omit in toc -->
```

Comment:
We agree to Meta filter to make the difference with the filters already in the rules.
A filter is in a rule , a Meta Filter is for many rules.